bindata/bootkube/config/bootstrap-config-overrides.yaml

```diff
@@ -14,16 +14,16 @@ extendedArguments:
   authorization-kubeconfig:
   - "/etc/kubernetes/secrets/kubeconfig"
   {{if .ClusterCIDR }}
-  cluster-cidr: {{range .ClusterCIDR}}
-  - {{.}}{{end}}
+  cluster-cidr:
+  - {{range $i, $cidr := .ClusterCIDR}}{{if $i}},{{end}}{{$cidr}}{{end}}
   {{end}}
   {{if .ServiceClusterIPRange }}
-  service-cluster-ip-range: {{range .ServiceClusterIPRange}}
-  - {{.}}{{end}}
+  service-cluster-ip-range:
+  - {{range $i, $ipRange := .ServiceClusterIPRange}}{{if $i}},{{end}}{{$ipRange}}{{end}}
```

Comment:
FWIW, cluster-kube-apiserver-operator just [does](https://github.com/openshift/cluster-kube-apiserver-operator/blob/151f2be4e47cc3826b266b16066829f3f4e16a98/bindata/bootkube/config/bootstrap-config-overrides.yaml#L24-L27):

```
{{- if .ServiceCIDR | len}}
servicesSubnet: {{index .ServiceCIDR 0 -}}
{{- if .ServiceCIDR | len | eq 2}},{{index .ServiceCIDR 1}}{{end -}}
{{- end}}
```

both `cluster-cidr` and `service-cluster-ip-range` are required to be no more than 2 elements.